### ISSUE TITLE
Black line running down the side of player

### ISSUE THREAD

**Issue by Pikachuy** [YePpHa/YouTubeCenter#1732]
![untitled](https://cloud.githubusercontent.com/assets/4439919/7440446/d9cdd2ea-f069-11e4-98e4-d3a9dcd4fcce.png)

This has happened before, both at the bottom or top. I'm assuming it came with the latest update. Any ideas on how to fix this?

**Comment by Yonezpt** [comment]
It is a YouTube layout problem, if you decrease your page zoom the black bar will increase in chunks.

**Comment by Pikachuy** [comment]
Yeah, I've toyed a bit more with the player size settings, and apparently this only happens with custom sizes, at least for me. It won't happen if I set the player size to 70% or 80%, but it will shrink and show the black line if I choose 75% (the player size I prefer).

**Comment by LiamBM** [comment]
![screen shot 2015-05-04 at 14 35 23](https://cloud.githubusercontent.com/assets/11802806/7454405/3f89fd7c-f26b-11e4-8c2a-b80a3888302a.png)

I've got 2 small lines at the top and bottom of the player.
